Interface/EmotionalCities.Pluma/EmotionalCities.Pluma.csproj

```diff
@@ -18,13 +18,13 @@
     <PackageOutputPath>..\bin\$(Configuration)</PackageOutputPath>
     <PackageTags>Bonsai Rx Emotional Cities Harp Pluma</PackageTags>
     <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
-    <VersionPrefix>0.1.0</VersionPrefix>
+    <VersionPrefix>0.1.1</VersionPrefix>
     <VersionSuffix></VersionSuffix>
-    <LangVersion>9.0</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Bonsai.Harp" Version="3.5.0" />
+    <PackageReference Include="OpenCV.Net" Version="3.4.2" />
```

Comment:
I think we would prefer to take a dependency on `System.Numerics` rather than on OpenCV for this one.

Reply:
This looks risky to change. Serialization wise both are structs one has Single (that is afterward defined has float) the other is a struct with floats only. It could be seamless, although lots of data with the previous one has already been collected and would rather keep things as used in field experiments.